commons-executors/src/main/java/com/palantir/common/concurrent/PTExecutors.java

```diff
@@ -295,10 +295,12 @@ public static ThreadPoolExecutor newFixedThreadPool(int numThreads, ThreadFactor
      * @throws IllegalArgumentException if <tt>numThreads &lt;= 0</tt>
      */
     public static ExecutorService newFixedThreadPool(int numThreads, String name) {
-        return MetricRegistries.instrument(
-                SharedTaggedMetricRegistries.getSingleton(),
-                PTExecutors.wrap(name, getViewExecutor(name, numThreads, Integer.MAX_VALUE, SHARED_EXECUTOR.get())),
-                name);
+        return MetricRegistries.executor()
+                .registry(SharedTaggedMetricRegistries.getSingleton())
```

Comment:
I know this is existing, it kills me to have this singleton in here and not inject registry